dispatches/models/renewables_case/wind_battery_double_loop_analysis.ipynb

```diff
@@ -0,0 +1,1237 @@
+{
```

Comment:
What is the point of this notebook?  It has no documentation.  Is it meant as the seed for future documentation, or the result of a particular analysis?  I think if it is meant to be included in the distribution, it should be moved into documentation (with significant explanatory text added)

Also, this has hard-coded paths to a developer's environment.  The notebook should be reworked to avoid that (so that users could actually run the notebook)